docs/source/advanced/apollo-ast.mdx

```diff
@@ -136,50 +127,54 @@ val schemaIssues = schemaGQLDocument.validateAsSchema()
 println(schemaIssues.map { it.severity.name + ": " + it.message })
 ```
 
-This will print `[WARNING: Unknown directive 'FooBar']`.
+When excuted, this snippet prints `[WARNING: Unknown directive 'private']`.
 
-The `validateAsExecutable` method ensures the operations are consistent with a schema, that is why it takes an instance
-of `Schema` as a parameter. This `Schema` can be obtained by calling the `GQLDocument.toSchema()` extension:
+### `validateAsExecutable`
+
+The `validateAsExecutable` method checks whether a document's defined operations are valid against a particular provided `Schema`. You can obtain this `Schema` parameter by calling `toSchema()` on the `GQLDocument` that represents the schema:
 
 ```kotlin
 val schema = schemaGQLDocument.toSchema()
 val executableIssues = queryGqlDocument.validateAsExecutable(schema)
 println(executableIssues.map { it.severity.name + ": " + it.message })
 ```
 
-This will print `[WARNING: Use of deprecated field 'height', 
-ERROR: Can't query 'foobar' on type 'Character']`
-
-## Transform the AST
+If the `queryGqlDocument` queries a deprecated field and misspells another, this snippet might print the following:
 
-The library also offers an API to manipulate the AST: the `transform` method on `GQLDocument`.
+```
+[WARNING: Use of deprecated field 'height', ERROR: Can't query 'frends' on type 'Character']
+```
 
-This method takes as a parameter a lambda that provides a `GQLNode` and returns the transformed node (or `null` to
-remove the node). It will then visit all nodes in the AST, apply the lambda to each one, and replace the node with the
-result.
+## Outputting SDL
 
-For instance, here's how to remove any field named `foobar` from the selections:
+You can output a `GQLDocument` to standard GraphQL SDL with the `toUtf8` extensions:
```

Comment:
Nit, I'd remove "SDL" up here as technically operations are not SDL (as they're not schemas obvisouly). Actually, I'm not exactly sure if SDL is something inside the spec.

```suggestion
You can output a `GQLDocument` to standard GraphQL with the `toUtf8` extensions:
```